lua/auto-session.lua

```diff
@@ -62,11 +64,28 @@ end
 
 
 ------ MAIN FUNCTIONS ------
-local AutoSession = {}
+local Config = {}
+
+function Config.normalize(config)
+  local conf = {}
+  if isEmptyTable(config) then
+    return conf
+  end
+
+  for k, v in pairs(config) do
+    conf[k] = v
+  end
+
+  return conf
+end
+
+local AutoSession = {
+  conf = {}
+}
 
 local function getSessionNameFromCwd()
   local cwd = vim.fn.getcwd()
-  return cwd:gsub("/", "-")
+  return cwd:gsub("/", "_")
```

Comment:
I wonder if there's an easy way to make this not break currently existing sessions using `_`.
I don't particularly mind using a dash or an underscore, but I wouldn't want to break people's existing sessions.

Reply:
On second thought I think neither dash nor underscore is the ideal character to use.
Probably better to go with the same thing Vim already uses for undo files, which is `%`.

Files would be `%Users%username%path%of%session.vim`

Reply:
Yeah, I agree with you, `%` sounds better. The reason for using underscore over dash is that leading dash is not very friendly for shell commands, e.g. deleting such files needs escaping.

Reply:
Yeah I realized the issues with `-` too late in the game 😅.
This looks great now, merging!

🙇